Automattic/wp-calypso
# Automattic for Agencies: Implement a stepper and header for the A4A Marketplace

Resolves https://github.com/Automattic/jetpack-genesis/issues/233

## Proposed Changes

This PR implements a stepper and header for the A4A Marketplace.

## Testing Instructions

Note: The steps are static currently, it doesn't change as of now.

- Switch branch: `git checkout add/a4a-marketplace-initial-layout`.
- Start the server by running `yarn start-a8c-for-agencies`.
- Click the `Marketplace` menu item > Verify that the page shows a stepper and a header with title and subtitle as shown below.

<img width="1251" alt="Screenshot 2024-02-21 at 1 00 24 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/5ef275dd-30ad-49bd-b166-8a550505e7b2">

- Append the URL with `?site_id={id}` and verify the site name is displayed

<img width="1251" alt="Screenshot 2024-02-21 at 1 00 15 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/191e76ec-7fd2-4010-b7bd-d127e252ea25">

## Pre-merge Checklist
